### ISSUE TITLE
Opening Annotation Note from Link Opens Editor

### ISSUE THREAD

**Issue by Oblique82** [elias-sundqvist/obsidian-annotator#128]
Even when "Use Annotation Mode by Default" is OFF - opening a note with an annotation target defaults to opening the PDF editor mode rather than the highlights in markdown mode

the expected behaviour occurs if the note is opened normally, but if it is a link from another note (or from a Dataview table) is still opened in PDF mode

currently using 0.2.0 beta

**Comment by aladmit** [comment]
Could you send a video with it, please? I tried a few times, but it didn't reproduce on my vault. Maybe, I understand something incorrectly from your description 🤔

**Comment by Oblique82** [comment]
sure sorry for the delay - here is the video. 

It shows that when the annotation note is opened from a link in another note it ALWAYS opens to annotation mode (regardless of the settings)

https://res.cloudinary.com/professional-integrative-medicine/video/upload/v1645148868/Troubleshooting/Screen_Recording_2022-02-18_at_12.14.15_pm_mzb8ax.mov

**Comment by aladmit** [comment]
Yeah, now I see what you mean. I've tried to reproduce it and found that work different depends on an editing mode. When I use Live Preview, link opens in md mode. When I use Source Mode, link opens in annotation mode.

Well, that's strange, but at least I understand how to reproduce it and why I didn't see this problem before 🥲

P.S. I tested on obsidian v0.13.26
